DataFormats/Luminosity/src/classes_def.xml

```diff
@@ -49,5 +49,9 @@
   <class name="reco::PixelClusterCounts" ClassVersion="3">
    <version ClassVersion="3" checksum="1474294271"/>
   </class>
- <class name="edm::Wrapper<reco::PixelClusterCounts>"/>
+ <class name="edm::Wrapper<reco::PixelClusterCountsPerEvent>"/>
+  <class name="reco::PixelClusterCountsPerEvent" ClassVersion="1">
+   <version ClassVersion="1"/>
```

Comment:
we should start from ```ClassVersion="3"``` https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning
The checksum is missing on L54.